features/gba-dev/kinderen.feature

```diff
@@ -0,0 +1,146 @@
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000048                       |
+      | fields              | kinderen.burgerservicenummer    |
+      Dan heeft de response een persoon met een 'kind' zonder gegevens
+
+    Scenario: ontkenning ouderschap
+      Gegeven de persoon met burgerservicenummer '000000061' heeft een 'kind' met de volgende gegevens
+      | voornamen (02.10) | voorvoegsel (02.30) | geslachtsnaam (02.40) | geboortedatum (03.10) | gemeente document (82.10) | datum document (82.20) | beschrijving document (82.30) | ingangsdatum geldigheid (85.10) | datum van opneming (86.10) |
+      | Daan              | de                  | Vries                 | 20031107              | 0518                      | 20031109               | PL gerelateerde               | 20031107                        | 20031109                   |
+      En het 'kind' is gewijzigd naar de volgende gegevens
+      | gemeente document (82.10) | datum document (82.20) | beschrijving document (82.30) | ingangsdatum geldigheid (85.10) | datum van opneming (86.10) |
+      | 1926                      | 20040105               | D27894-2004-A782              | 20031107                        | 20040112                   |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000061                       |
+      | fields              | kinderen                        |
+      Dan heeft de response een persoon zonder gegevens
+
+    Scenario: met fields vragen om een gegeven zonder waarde
+      Gegeven de persoon met burgerservicenummer '000000073' heeft een 'kind' met de volgende gegevens
+      | voornamen (02.10) | geslachtsnaam (02.40) |
+      | Bert              | Jansen                |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000073                       |
+      | fields              | kinderen.burgerservicenummer    |
+	  Dan heeft de response een persoon zonder gegevens
+
+  Rule: de geleverde kindgegevens zijn de gegevens zoals die staan op de persoonslijst van de gevraagde persoon
+    Bij het raadplegen van een persoon worden alleen gegevens uit de persoonslijst van de gevraagde persoon gebruikt, en nooit gegevens van de persoonslijst van het kind
+
+    Scenario: Het kind heeft geslachtswijziging en naamswijziging ondergaan, maar de gevraagde persoon erkent dit niet
+      Gegeven de persoon met burgerservicenummer '000000085' heeft een 'kind' met de volgende gegevens
```

Comment:
hier geldt hetzelfde als in #1257 staat over regel 130. Als je niet ook de PL van het kind opneemt in Gegeven toont (en test) het scenario niet wat wat het hoort te tonen/testen

Reply:
Impliciet technisch wel, maar we maken toch features om functionaliteit expliciet te maken?

Reply:
Voorstel is een extra Gegeven op te nemen waarmee de PL van het kind wordt opgenomen in de database. Dit kind wordt vervolgens ook gewijzigd. Deze gegevens stappen worden vooralsnog uitbecommentarieerd omdat de automation code dit nog niet ondervangt. 

Dit moet tevens in de gba-dev variant worden aangebracht.